### DISCUSSION TITLE
Species Randomization

### DISCUSSION THREAD

**Discussion by ScottMacek** [bcholmes/StarTrek2d20#186] [General]
Please create a random option for all available species based on available resources!  I appreciate the Core Book options are available randomly, but I would love a random generator that you can pick and choose which species to allow and maybe adjust how frequently different options occur.  Worst case, even just returning the Alpha Quadrant and Beta Quadrant random tables would be a big help!  Other than this, I still love the character creation tool and look forward to whatever improvements you have in mind down the road!

**Comment by bcholmes** [comment]
You'll be happy to know that this idea fit in with a change I was planning already. The version of the app now available has that ability.

**Reply by ScottMacek** [reply]
Sigh.  If I weren't already happily married... thank you!

**Reply by Felderburg** [reply]
The new randomization dropdown is definitely a good look! I wonder, though, if there should be an "other" option—for things that aren't in the core/quadrant books. Specifically things like androids from crew packs, or kelpians from the Disco campaign.
